_data/config.yml

```diff
@@ -1,2 +1,3 @@
 grpc_release_branch: release-0_14
+grpc_java_release_tag: v1.0.0-pre1
```

Comment:
Fair enough. This would also be a good test that ensures that hard coded references to 1.0 now point to this variable instead.